ci/taos/plugins-base/pr-audit-build-ubuntu.sh

```diff
@@ -105,7 +105,11 @@ function pr-audit-build-ubuntu-run-queue(){
         # --use-pdebuild-internal runs "debian/rules clean"inside the chroot
         # http://pbuilder-docs.readthedocs.io/en/latest/usage.html
         # example: sudo -Hu www-data pdebuild
-        pdebuild  --use-pdebuild-internal 2> ../report/build_log_${input_pr}_ubuntu_error.txt 1> ../report/build_log_${input_pr}_ubuntu_output.txt
+        echo -e "[DEBUG] pdebuild start at :"
+        date -u
```

Comment:
Please use -R instead of -u. we need RFC 2822.
```
  -R, --rfc-2822
              output date and time in RFC 2822 format.  Example: Mon, 14 Aug 2006 02:34:56 -0600
```